tests/test_client.py

```diff
@@ -0,0 +1,382 @@
+import json
+import math
+import os
+import runpy
+import subprocess
+import sys
+import time
+import uuid
+from typing import Any, Optional, TypedDict, cast
+
+import sqlalchemy as sa
+
+from dbos import DBOS, ConfigFile, DBOSClient, EnqueueOptions, Queue, SetWorkflowID
+from dbos._dbos import WorkflowHandle
+from dbos._schemas.system_database import SystemSchema
+from dbos._sys_db import SystemDatabase
+from dbos._utils import GlobalParams
+from tests.client_collateral import event_test, retrieve_test, send_test
+
+
+class Person(TypedDict):
+    first: str
+    last: str
+    age: int
+
+
+def run_client_collateral() -> None:
+    dirname = os.path.dirname(__file__)
+    filename = os.path.join(dirname, "client_collateral.py")
+    runpy.run_path(filename)
+
+
+def test_client_enqueue_appver_not_set(dbos: DBOS, client: DBOSClient) -> None:
+    run_client_collateral()
+
+    johnDoe: Person = {"first": "John", "last": "Doe", "age": 30}
+    wfid = str(uuid.uuid4())
+
+    options: EnqueueOptions = {
+        "queue_name": "test_queue",
+        "workflow_name": "enqueue_test",
+        "workflow_id": wfid,
+    }
+
+    client.enqueue(options, 42, "test", johnDoe)  # type: ignore
```

Comment:
What's wrong with the types?

Reply:
Still working thru the typing

Reply:
Moved the `#type: ignore` declaration to `enqueue` so the caller doesn't have to specify it